### PR TITLE
[WIP] Added CSV Support

### LINKED ISSUE #210: Addition of  CSV support
- [x] Feature Request

#### Current Behaviour
As of now, result supports only JSON, XML.

#### Expected Behaviour
Result should support JSN, XML, CSV 

**Yes, I would like to work on this issue**

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #210 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -47,8 +47,11 @@
                                         <input type="radio" value="json" autocomplete="off">JSON
                                     </label>
                                 <label class="btn btn-lg btn-primary formatButton">
-                                        <input type="radio" value="xml" autocomplete="off">XML
+                                        <input type="radio" value="csv" autocomplete="off">CSV
```
Comment (bhaveshAn) [inline app/templates/index.html]: Why to replace xml with csv here ?

File: app/templates/index.html [hunk 2]
```diff
@@ -138,8 +141,12 @@
                             if (sformat == "json") {
                                 $(".code").attr("class", "language-json");
                                 response = JSON.stringify(response, null, 4);
-                            } else
+                            } else if (sformat == "xml"){
                                 response = new XMLSerializer().serializeToString(response);
+                            }else{
+                                console.log("CSV Respose here");
+                                return response;
```
Comment (bhaveshAn) [inline app/templates/index.html]: Please look into it. Is really the csv response will be returned like as normal ?

File: app/server.py [hunk 3]
```diff
@@ -5,6 +5,7 @@
 from xml.dom.minidom import parseString
 import json
 import os
+import csv
```
Comment (bhaveshAn) [inline app/server.py]: Not used csv module in the code below.

File: app/templates/index.html (hunk 2 above)
Comment (vaibhavsingh97) [reply app/templates/index.html]: Nope, This PR is not complete

File: app/server.py (hunk 3 above)
Comment (vaibhavsingh97) [reply app/server.py]: This PR is not complete

**Comment by ParthS007** [comment]
closing this due to inactivity

**Pull request closed without merging**